frontend/src/metabase/components/MetadataInfo/DimensionInfoPopover/DimensionInfoPopover.tsx

```diff
@@ -39,7 +40,7 @@ function DimensionInfoPopover({
   return hasMetadata ? (
     <TippyPopover
       className={className}
-      delay={isCypressActive ? 0 : POPOVER_DELAY}
+      delay={isCypressActive ? 0 : delay}
```

Comment:
Maybe we should do this inside `TippyPopover`? Probably, all components that use it with a delay will require making it zero for tests

Reply:
yeah good observation, moved that logic to `TippyPopover`